src/main/app/src/components/common/Header.js

```diff
@@ -114,7 +123,10 @@ export const Header = (props) => {
             onClick={toggleMenu}
             edge="start"
             className={classes.menuButton}>
-            {isOpen ? <Icon>close</Icon> : <MenuIcon />}
+            <Box display="flex" className={classes.menuBox}>
```

Comment:
Redundantti display-prop kun on jo annettu CSS:ssä.
```suggestion
            <Box className={classes.menuBox}>
```